workflows/gas_nomenclature.nf

```diff
@@ -62,11 +78,47 @@ workflow GAS_NOMENCLATURE {
     }
 
     reference_values = profiles.ref.collect{ meta, profile -> profile}
-    query_values = profile.query.collect{ meta, profile -> proifile }
-    reference_values.view()
-    query_values.view()
-    //LOCIDEX_MERGE_REF(reference_values)
-    //LOCIDEX_MERGE_QUERY(query_values)
+    query_values = profiles.query.collect{ meta, profile -> profile }
+
+    // LOCIDEX modules
+    ref_tag = Channel.value("ref")
+    query_tag = Channel.value("value")
+    merged_references = LOCIDEX_MERGE_REF(reference_values, ref_tag)
+    ch_versions = ch_versions.mix(merged_references.versions)
+
+    merged_queries = LOCIDEX_MERGE_QUERY(query_values, query_tag)
+    ch_versions = ch_versions.mix(merged_queries.versions)
+
+
+    // PROFILE DISTS processes
+
+    mapping_file = prepareFilePath(params.pd_mapping_file)
+    if(mapping_file == null){
+        exit 1, "${params.pd_mapping_file}: Does not exist but was passed to the pipeline. Exiting now."
```

Comment:
Do you know if these are the accepted way of exiting from nextflow, or should we use `error()`?